src/components/Select/Select.styles.js

```diff
@@ -1,14 +1,31 @@
 import styled from 'styled-components';
 
-export const SelectWrapper = styled.div`
+export const SelectTag = styled.select`
+  cursor: pointer;
+  color: var(--darkgrey);
+  font-size: 1.6rem;
+
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+`;
+
+export const LabelTag = styled.label`
+  font-weight: bold;
+  color: var(--darkblue);
+`;
+
```

Comment:
Delete this code - not needed. The SelectTag styling is duplicating select styling below.
LableTag styling is already defined in typography.styles.js